docs/source/api/apollo-server.mdx

```diff
@@ -599,6 +599,69 @@ The `start` method triggers the following actions:
 1. If your server is a [federated gateway](https://www.apollographql.com/docs/federation/managed-federation/overview/), it attempts to fetch its schema. If the fetch fails, `start` throws an error.
 2. Your server calls all of the [`serverWillStart` handlers](../integrations/plugins-event-reference/#serverwillstart) of your installed plugins. If any of these handlers throw an error, `start` throws an error.
 
+
+## `executeOperation`
+
+The async `executeOperation` method is used primarily for [testing GraphQL operations](../testing/testing/#testing-using-executeoperation) through the server's request pipeline without going through a full HTTP operation.
+
+The `executeOperation` method takes two arguments:
+* The first argument is a configuration options object
+* The second argument will be passed in to the current `ApolloServer` instance's `context` function
```

Comment:
"will be passed in to" -> "is passed to"

"the current `ApolloServer` instance's" -> "Apollo Server's" (I think it's clear enough that this is a function associated with a particular instance)

Reply:
We should probably have a child bullet here that explains why passing a value to the `context` function is helpful. Could just link to the conceptual article for details.